go-controller/pkg/ovn/ovn.go

```diff
@@ -271,6 +292,19 @@ func (oc *DefaultNetworkController) removeRemoteZonePod(pod *kapi.Pod) error {
 				getPodNamespacedName(pod), err)
 		}
 	}
+	if kubevirt.IsPodLiveMigratable(pod) {
+		// After live migration to a different zone ip should be deallocated
+		// from remote zone if VM is gone
+		if err := oc.deleteLogicalPort(pod, portInfo); err != nil {
+			return fmt.Errorf("deleteLogicalPort failed for VM pod %s: %w at remote zone",
+				getPodNamespacedName(pod), err)
+		}
```

Comment:
This is doing **a lot** more than just releasing IPs. Can you do something more specific?

Reply:
Also this belongs to the second commit

Reply:
After some talk we found that `deleteLogicalPort` should be idempotent so is ok to call it here again, also I will move it to second commit.